EventLite/Data/CatalogContext.cs

```diff
@@ -32,9 +32,6 @@ public class CatalogContext : DbContext
         // How
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            // QUESTION:
```

Comment:
Please keep this comment, I will investigate

Reply:
Added back